bigbluebutton-html5/public/locales/en.json

```diff
@@ -624,6 +628,8 @@
     "app.settings.dataSavingTab.webcam": "Enable other participants webcams",
     "app.settings.dataSavingTab.screenShare": "Enable other participants desktop sharing",
     "app.settings.dataSavingTab.description": "To save your bandwidth adjust what's currently being displayed.",
+    "app.settings.transcriptionTab.partialUtterances": "Show partial utterances",
```

Comment:
@lfzawacki Have you considered renaming the labels to be more descriptive? For example: "Display the interim transcription (when disabled we will only display results that have been refined by analyzing the full sentence)"